alphaSat-HMMER/inputs.json

```diff
@@ -0,0 +1,6 @@
+{ 
```

Comment:
Is this file is conflicting with this file [alphaSat-HMMER-local.json](https://github.com/kmiga/alphaAnnotation/blob/CenSat/alphaSat-HMMER/test/alphaSat-HMMER-local.json)?

Reply:
I'm not sure - it looks like alphaSat-HMMER-local.json is missing the sample_id input?